camkes/templates/rpc-connector-common-from.c

```diff
@@ -54,7 +54,7 @@
 
     unsigned size;
     /*-- if m.return_type is not none -*/
-      /*? macros.show_type(m.return_type) ?*/ return_val;
+      /*? macros.show_type(m.return_type) ?*/ return_val = { 0 };
```

Comment:
I believe this change means that the 2 memset calls generated later in this function are now longer necessary.  Are you able to remove them?

Reply:
Hey @kent-mcleod. I wasn't sure if a failing marshalling would clobber the buffer or not, that's why I left them in. But after your comment I've updated the PR.

Reply:
Yes you're right that a failing clobber could leave the buffer in an undefined state and should be cleared before returning.
Sorry, I originally realized my mistake a few minutes after making my original comment, but by the time I got back to the computer I forgot to come back and update.

Reply:
No worries. I reverted the change. But it looks like you also pushed a change.

Reply:
Ah yes.  I just pushed it back to the previous version to save you the trouble before merging.